test/libyul/functionSideEffects/empty_with_sstore.yul

```diff
@@ -2,4 +2,4 @@
     sstore(0, 1)
 }
 // ----
-// : invalidatesStorage
+// : writes other state, writes storage
```

Comment:
Which other state does it write?

Reply:
Was a typo in `otherState` (SSTORE was considered as Write.) Fixed this.